things/policies/service.go

```diff
@@ -178,3 +213,29 @@ func (svc service) hasThing(ctx context.Context, p Policy) (string, error) {
 	}
 	return thingID, nil
 }
+
+func (svc service) identify(ctx context.Context, token string) (string, error) {
+	req := &upolicies.Token{Value: token}
+	res, err := svc.auth.Identify(ctx, req)
+	if err != nil {
+		return "", errors.Wrap(errors.ErrAuthorization, err)
+	}
+	return res.GetId(), nil
+}
+
+func (svc service) checkAdmin(ctx context.Context, id string) error {
+	req := &upolicies.AuthorizeReq{
+		Sub:        id,
+		Obj:        thingsObjectKey,
+		Act:        "c_update", // TODO: remove since this is not used
```

Comment:
Do we need this TODO?

Reply:
Yes. @dborovcanin had suggested we move admin check to `JWT` layer so that we have role information in the `JWT` token